api/controllers/AuthController.js

```diff
@@ -60,10 +60,48 @@ module.exports = {
           res.view(403, {error: err});
           return;
         }
-        if (url) {
-          return res.redirect(url);
-        }
-        return res.redirect('/');
+
+        Reddit.checkModeratorStatus(
```

Comment:
One thing we need to discuss (I suggest we take some proper time to discuss with Raia/TSAR as well) is this means that a moderator, after stepping down (or the unlikely scenario of being forcible removed) will still be a fapp moderator until they log out.

How do we want to deal with this? Options:

1. Just clear the user's session information when they step down. I can do this manually or:
2. Make a button to clear all the mod's sessions that any mod can press
3. Send off a background process to check the moderator status every so often (week/day/hour?)